.github/workflows/docker.yml

```
@@ -1,25 +1,39 @@
 name: Build & Push Docker image
+
 on:
   push:
     branches:
       - master
     paths:
       - 'APPVERSION'
+  # Allow manual runs.
+  workflow_dispatch:
+
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
       - name: checkout code
         uses: actions/checkout@v2
-      - name: install buildx
-        id: buildx
-        uses: crazy-max/ghaction-docker-buildx@v1
+
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v1.0.1
+
+      - name: Setup Buildx
+        uses: docker/setup-buildx-action@v1.0.3
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1 
         with:
-          version: latest
-      - name: login to docker hub
-        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USER }}" --password-stdin
-      - name: build image
-        run: |
-        docker buildx build - push \
-          --tag moncho/dry:latest \
-          --platform linux/amd64,linux/arm/v7,linux/arm64 .
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v2.1.0
+        with:
+          context: .
+          file: ./Dockerfile
+          tags: florianpiesche/dry:latest
```

Would you mind changing it to **moncho/dry**? It is unofficially the official Docker image for **dry**.

```suggestion
          tags: moncho/dry:latest
```